pkg/init/init

```diff
@@ -1,9 +0,0 @@
-#!/bin/sh
-
-/bin/mount -t tmpfs tmpfs /mnt
-
-/bin/cp -a / /mnt 2>/dev/null
-
-/bin/mount -t proc -o noexec,nosuid,nodev proc /proc
```

Comment:
your go code does not mount `/proc` is that only mounted here for the `switch_root`?